tools/gdb/README.md

```diff
@@ -0,0 +1,12 @@
+# Custom gdb scripts
+Custom gdb scripts for gdb debugging and autoloading of these scripts.
+individual scripts are located in the `gdb_scripts/` and the top level directory is reserved for loader scripts `load_gdb_scripts.py`.
+
+### Usage
+To loads scripts from within gdb call `source /path/to/load_gdb_scripts.py`.
+If you have compiled WiredTiger with the `-DENABLE_SHARED=1` flag then these scripts 
+will be auto-loaded when gdb is opened.
+
+### Creating new scripts
+Scripts can be written in either python ([hazard_pointers.py](./gdb_scripts/hazard_pointers.py)) or scheme ([dump_row_int.gdb](./gdb_scripts/dump_row_int.gdb)). If you create a new python script it is recommended you follow the pattern of `hazard_pointers.py` where a class that extends `gdb.Command` is created for each command. Don't forget to register the new command by calling its constructor at the bottom of the file.
```

Comment:
What is meant by "scheme"? They're called "command files" [here](https://sourceware.org/gdb/current/onlinedocs/gdb.html/Command-Files.html), we should use that term.

Reply:
Another mistake on my end. GDB [can be extended](https://sourceware.org/gdb/current/onlinedocs/gdb.html/Extending-GDB.html#Extending-GDB) using Guile (an implementation of the scheme language) instead of python, but command file is the correct term.
This and the above typos have been fixed in [9d39201](https://github.com/wiredtiger/wiredtiger/pull/9801/commits/9d392011cc33bf58e1cb742532691752cdcfe057)